content/en/docs/concepts/services-networking/topology-aware-routing.md

```diff
@@ -145,6 +168,11 @@ zone.
   proportions of each zone. This could have unintended consequences if a large
   portion of nodes are unready.
 
+* The EndpointSlice controller ignores nodes with the
+  `node-role.kubernetes.io/control-plane` or `node-role.kubernetes.io/master`
```

Comment:
Optionally, link to the definitions of these labels in https://kubernetes.io/docs/reference/labels-annotations-taints/

Reply:
Ran out of time for this one, not quite sure how to make this look right and don't want to delay the rest of the PR.

Reply:
Created followup issue for this: https://github.com/kubernetes/website/issues/40602